### DISCUSSION TITLE
How to implepent.

### DISCUSSION THREAD

**Discussion by Umerjamshaid** [lowlighter/metrics#1596] [Q&A] [answered]
How do I implement badges, calendars & etc, into my `readme. me`.
   How to do it, I am trying to build my git profile as professional as it is.

**Comment by jayvdb** [comment, marked as answer]
Contribute to some projects.  See https://github.com/firstcontributions/first-contributions for how to get started.

**Reply by Umerjamshaid** [reply]
thanks, brother😁😁
